pkg/config/default.go

```diff
@@ -43,26 +43,29 @@ const (
 	// _defaultTransport is a prefix that we apply to an image name to check
 	// docker hub first for the image.
 	_defaultTransport = "docker://"
+
+	// _defaultImageVolumeMode is a mode to handle built-in image volumes.
+	_defaultImageVolumeMode = "bind"
 )
 
 var (
-	// DefaultInitPath is the default path to the container-init binary
+	// DefaultInitPath is the default path to the container-init binary.
 	DefaultInitPath = "/usr/libexec/podman/catatonit"
-	// DefaultInfraImage to use for infra container
+	// DefaultInfraImage to use for infra container.
 	DefaultInfraImage = ""
-	// DefaultRootlessSHMLockPath is the default path for rootless SHM locks
+	// DefaultRootlessSHMLockPath is the default path for rootless SHM locks.
 	DefaultRootlessSHMLockPath = "/libpod_rootless_lock"
 	// DefaultDetachKeys is the default keys sequence for detaching a
-	// container
+	// container.
 	DefaultDetachKeys = "ctrl-p,ctrl-q"
 	// ErrConmonOutdated indicates the version of conmon found (whether via the configuration or $PATH)
-	// is out of date for the current podman version
+	// is out of date for the current podman version.
 	ErrConmonOutdated = errors.New("outdated conmon version")
-	// ErrInvalidArg indicates that an invalid argument was passed
+	// ErrInvalidArg indicates that an invalid argument was passed.
 	ErrInvalidArg = errors.New("invalid argument")
```

Comment:
All of these errors will have stack traces in them that point to this location, because they're being created by the `github.com/pkg/errors` package and not the standard library's `errors` package.

Reply:
If they're being wrapped, it'll save people who troubleshoot these errors a bit of time because they won't start looking in the wrong place for the code that triggered error.  If not, then no.

Reply:
I added a patch to convert pkg/config to use "errors" all the time.